packages/evm/src/precompiles/05-modexp.ts

```diff
@@ -134,9 +134,10 @@ export function precompile05(opts: PrecompileInput): ExecResult {
   if (!opts.common.isActivatedEIP(2565)) {
     gasUsed = (adjustedELen * multiplicationComplexity(maxLen)) / Gquaddivisor
   } else {
-    gasUsed = (adjustedELen * multiplicationComplexityEIP2565(maxLen)) / Gquaddivisor
-    if (gasUsed < BIGINT_200) {
-      gasUsed = BIGINT_200
+    const wordsSquared = multiplicationComplexityEIP2565(maxLen)
```

Comment:
This should be changed only iff EIP 7883 is active. Now the changes with EIP-2565 active reflect what should be active when EIP 7883 is active.

Common should include a new EIP, and this `if` statement should check if EIP-7883 is active (run the new schedule), or if EIP-2565 is active (if EIP-7883 is thus not active) then run that logic. If EIP-2565 is also not active, then run the legacy (pre-EIP-2565) gas costs.

This should also fix the failing state/blockchain tests.